Kwf/Media.php

```diff
@@ -199,7 +199,8 @@ private static function _getOutputWithoutCheckingIsValid($class, $id, $type)
             if ($useCache) {
                 if (isset($output['contents']) && strlen($output['contents']) > 20*1024) {
                     //don't cache contents larger than 20k in apc, use separate file cache
-                    $cacheFileName = Kwf_Config::getValue('mediaCacheDir').'/'.$class.'/'.$id.'/'.$type;
+                    $similarIdsSameFolder = substr_replace($id, '/', 2, 0);
```

Comment:
Don't take the fist part of the ID since it's often a componentId which starts with "root". Better is to create to use a subfolder which is named like substr(md5($id), 0, 2).